mdoc/Test/html.expected/System/Array.html

```diff
@@ -293,6 +293,15 @@ <h2 class="Section">Extension Methods</h2>
                 </td>
                 <td colspan="2">
                   <b>
+                    <a href="../System/Environment.html#M:System.Environment.IsAligned``1(``0[],System.Int32)">IsAligned&lt;T&gt;</a>
```

Comment:
@jonpryor see above ... the method correctly shows now in the list of extension methods in the Array html.